### PR TITLE
Feature: Updates for coordinator, tx-processor and zki initial implementation

### PR DESCRIPTION
**Pull request by Shailu-s**
This PR contains - 

- Updates for txprocessor for processing a transactions, removed txs inputs which were not required.
- Updated zki based on the requirements, Updated func to initialize zki.
- Updated logic of running txprocessor as batchbuilder to update zki.

### CODE REVIEW

File: go_sequencer/database/kvdb/kvdb.go [hunk 1]
```diff
@@ -219,6 +222,7 @@ func (k *KVDB) reset(batchNum common.BatchNum, closeCurrent bool) error {
 		}
 		k.db = sto
 		k.CurrentAccountIdx = common.RollupConstReservedIDx // 255
+		k.CurrentVouchIdx = common.RollupConstReservedIDx
```
Comment (kaiden-araki) [inline go_sequencer/database/kvdb/kvdb.go:226]: @Shailu-s Is `CurrentVouchIdx` also reset as `RollupConstReservedIDx`(255) when kvdb is reset?

File: go_sequencer/test/til/txs.go [hunk 2]
```diff
@@ -167,27 +167,7 @@ func (tc *Context) generateBlocks() ([]common.BlockData, error) {
...
-			testTx := L1Tx{
-				lineNum:     inst.LineNum,
-				fromIdxName: inst.From,
-				toIdxName:   inst.To,
-				L1Tx:        tx,
-			}
-			if err := tc.addToL1UserQueue(testTx); err != nil {
-				return nil, common.Wrap(err)
-			}
+		//Removed case for TxTypeCreateAccountDepositTransfer
```
Comment (kaiden-araki) [inline go_sequencer/test/til/txs.go]: @Shailu-s This block is essential for `TxTypeCreateAccountDeposit` case.

File: go_sequencer/common/batch.go [hunk 3]
```diff
@@ -8,37 +8,35 @@ import (
...
+	EtherPriceUSD float64           `meddler:"ether_price_usd"`
 	// ForgeL1TxsNum is optional, Only when the batch forges L1 txs. Identifier that corresponds
 	// to the group of L1 txs forged in the current batch.
 	ForgeL1TxsNum *int64   `meddler:"forge_l1_txs_num"`
 	SlotNum       int64    `meddler:"slot_num"` // Slot in which the batch is forged
 	TotalFeesUSD  *float64 `meddler:"total_fees_usd"`
 }
 
-type BatchNum int64
+type BatchNum uint32
```
Comment (Intizar-T) [inline go_sequencer/common/batch.go:34]: I'm curious why we are changing this. Is the assumption we won't reach `4294967295` batch number, which is the largest number `uint32` can hold?

Comment (Shailu-s) [reply go_sequencer/common/batch.go:34]: It's on the basis of the struct doc shared here - [Link](https://www.notion.so/tokamak/Rollup-Circuit-Specification-fa2729166fb34a91b5fe8dcd4b0dcf40)

**Comment by Shailu-s** [comment]
This PR changes are included in [PR-60](https://github.com/tokamak-network/tokamak-sybil-resistance/pull/60/files). Hence closing this one.

**Pull request closed without merging**